luaui/Widgets/gui_gridmenu.lua

```diff
@@ -810,6 +815,24 @@ function reloadBindings()
 	else
 		Spring.SendCommands("bind " .. string.lower(PREV_PAGE_KEY) .. " gridmenu_prev_page")
 	end
+
+        copyKeyLayout(currentLayout, 'custom')
```

Comment:
Tabs!

I actually prefer spaces over tabs, but we want to use the convention for this project.